app/controllers/comments_controller.rb

```diff
@@ -176,7 +200,11 @@ def create
             cookies[:comment_name] = @comment.name[0..100]
             cookies[:comment_email] = @comment.email[0..100]
           end
-          flash[:comment_notice] = ts('Comment created!')
+          if @comment.unreviewed?
+            flash[:comment_notice] = ts("Your comment was received! It will appear publicly after the author has approved it.")
```

Comment:
This should say "creator" or "work creator" to avoid assuming the work is something written.